identity/aziot-identityd/src/settings.rs

```diff
@@ -9,17 +9,15 @@ use crate::error::InternalError;
 
```

Comment:
Is there a reason this file is called settings.rs in identityd but config.rs in certd/keyd? I feel like the file names should be consistent.

Reply:
This is a hangover from iotedged using settings.rs for parsing YAML. We can rename this to be consistent.

Reply:
@arsing - are you renaming this? If not, I can do this in the identityd UDS PR.

Reply:
I'm not.

We should probably consolidate some of the "entrypoint" stuff, like reading env vars for configs, env_logger, etc. But I talked with Cindy about a thing that would affect this anyway. Let's talk offline.